src/parse_utils.rs

```diff
@@ -88,19 +88,16 @@ pub fn next_token(
 {
 	match iter.next()
 	{
-		Some(TokenTree::Group(group)) if group.delimiter() == Delimiter::None =>
+		Some(TokenTree::Group(ref group)) if group.delimiter() == Delimiter::None =>
 		{
 			let mut in_group = group.stream().into_iter();
 			let result = in_group.next();
-			match in_group.next()
+			match (in_group.next(), in_group.next())
 			{
-				None => Ok(result),
+				(None, None) => Ok(result),
```

Comment:
```suggestion
				(None, _) => Ok(result),
```
Using `_` more accurately maintains semantics and conveys that the second token is irrelevant in the first is `None`

Reply:
This has been applied manually. I split off this function from the rest of the bind-by-move changes into its own commit, as it is a change a bit more invasive than the others.